src/main/java/gift/repository/WishRepository.java

```diff
@@ -11,16 +12,16 @@
 @Repository
 public interface WishRepository extends JpaRepository<Wish, Long> {
 
-    List<Wish> findByUserId(Long userId);
+    Optional<List<Wish>> findByUserId(Long userId);
```

Comment:
List를 반환하는 경우 null을 반환하지 않아요. Optional로 감싸지 않고 사용합니다.
```suggestion
    List<Wish> findByUserId(Long userId);
```